geanypg/src/geanypg.c

```diff
@@ -58,7 +57,7 @@ gpgme_error_t geanypg_show_err_msg(gpgme_error_t err)
     gchar const * msg = (gchar const *)gpgme_strerror(err);
     gchar const * src = (gchar const *)gpgme_strsource(err);
     dialogs_show_msgbox(GTK_MESSAGE_ERROR, "%s %s: %s\n", _("Error from"), src, msg);
-    fprintf(stderr, "GeanyPG: %s %s: %s\n", _("Error from"), msg, src);
+    g_error("%s %s: %s", _("Error from"), msg, src);
```

Comment:
don't use [`g_error()`](https://developer.gnome.org/glib/unstable/glib-Message-Logging.html#g-error) as it `abort()`s.